taca/cleanup/cleanup.py

```diff
@@ -0,0 +1,644 @@
+                logger.info("Removing fastq_files from {}".format(proj_data_root))
+                if not dry_run:
+                    _remove_files(fastq_proj['fastq_files'])
+                    logger.info("Removed fastq files from data directory for project {}, marking it as cleaned".format(proj))
+                    _touch_cleaned(proj_data_root)
+            
+        analysis_info = info.get('analysis_to_remove')
+        if analysis_info and isinstance(analysis_info, dict):
+            proj_analysis_root = analysis_info['proj_analysis_root']
+            logger.info("cleaning analysis data for project {}".format(proj))
+            for qc, files in analysis_info['analysis_files'].iteritems():
+                removed_qc = []
+                logger.info("Removing files of '{}' from {}".format(qc, proj_analysis_root))
+                if not dry_run:
+                    _remove_files(files)
+                    removed_qc.append(qc)
+            map(analysis_info['analysis_files'].pop, removed_qc)
+            if len(analysis_info['analysis_files']) == 0:
+                logger.info("Removed analysis data for project {}, marking it cleaned".format(proj))
+                _touch_cleaned(proj_analysis_root)
+
+
+#############################################################
+# Class helper methods, not exposed as commands/subcommands #
+#############################################################
+
+def get_closed_proj_info(pdoc):
+    """check and return a dict if project is closed"""
+    pdict = None
+    if "close_date" in pdoc:
+        closed_date = pdoc['close_date']
+        closed_days = misc.days_old(closed_date, "%Y-%m-%d")
+        pdict = {'name' : pdoc.get('project_name'),
+                 'pid' : pdoc.get('project_id'),
+                 'closed_date' : closed_date,
+                 'closed_days' : closed_days,
+                 'bioinfo_responsible' : pdoc.get('project_summary',{}).get('bioinfo_responsible','')}
+    return pdict
+
+def collect_analysis_data_irma(pid, analysis_root, files_ext_to_remove={}, return_size=False):
+    """Collect the analysis files that have to be removed from IRMA"""
+    proj_abs_path = os.path.join(analysis_root, pid)
+    if not os.path.exists(proj_abs_path):
+        file_list = None
+    elif os.path.exists(os.path.join(proj_abs_path, "cleaned")):
+        file_list = "cleaned"
+    else:
+        file_list = {'proj_analysis_root':proj_abs_path,
+                     'analysis_files': defaultdict(list)}
+        for qc_type,ext in files_ext_to_remove.items():
+            qc_path = os.path.join(proj_abs_path, qc_type)
+            if os.path.exists(qc_path):
+                file_list['analysis_files'][qc_type].extend(collect_files_by_ext(qc_path, ext))
+    if return_size:
+        size = 0
+        try:
+            size += sum([sum(map(os.path.getsize, fls)) for fls in file_list['analysis_files'].values()])
+        except:
+            pass
+        return (file_list, size)
```

Comment:
@senthil10 I am not a big fun of functions returning different amounts of outputs... even if we are not intrested in `return_size ` you should return it so the last line should always be `return (file_list, size)` and you can initialise the variable `size` outside the if